### PR TITLE
OEL-2604: Change alignment of the sorting options

### PR DESCRIPTION
**Pull request by AaronGilMartinez**

### CODE REVIEW

**Review by drishu — CHANGES_REQUESTED** [review]
I dont think a change in the template is needed, I think its rather a css issue.
If I remove `.oe-list-pages-sort-form .form-item{margin:0 0 0 1.5rem}` for instance then it looks ok above the 1200px breakpoint
And then some work on the sort label's and select's margins are needed.

On other list pages its ok because the sort option are smaller, so what if we get much longer ones?

**Review by drishu — DISMISSED** [review]

File: resources/sass/components/_list_pages.scss [hunk 1]
```diff
@@ -17,6 +17,12 @@
   margin-right: 1.5rem;
 }
 
+.oe-list-pages-sort-form .form-select {
+  @media (min-width: 576px) {
```
Comment (brummbar) [inline resources/sass/components/_list_pages.scss]: This is not working:
image

**Review by brummbar — CHANGES_REQUESTED** [review]

File: resources/sass/components/_list_pages.scss [hunk 2]
```diff
@@ -1,39 +1,22 @@
...
-  margin-right: 1.5rem;
-}
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
 
-.oe-list-pages-sort-form .form-select {
-  @media (min-width: 576px) {
-    max-width: calc(100% - 75px);
+  .form-label {
```
Comment (drishu) [inline resources/sass/components/_list_pages.scss]: dont think this is ok, and its worse for me now, see
Screenshot 2024-06-05 at 18 15 43

**Review by drishu — CHANGES_REQUESTED** [review]

**Review by drishu — DISMISSED** [review]
The solution looks better now

**Comment by brummbar** [comment]
Still has some issues on some viewports:
image

**Review by AaronGilMartinez — COMMENTED** [review]
Approving, can't approve my own PR

**Review by brummbar — APPROVED** [review]

**Pull request merged**